Listenbrainz/UI/Components/SettingsView.swift

```diff
@@ -10,10 +10,11 @@ import SwiftUI
 
 struct SettingsView: View {
   @AppStorage("isDarkMode") private var isDarkMode = false
-  @AppStorage("userToken") private var userToken:String = ""
+  @State  var userToken: String
   @State private var userName:String = ""
```

Comment:
Also keep this in the app storage for now

Reply:
done